src/MakePotCommand.php

```diff
@@ -618,6 +618,17 @@ protected function extract_strings() {
 				PhpCodeExtractor::fromDirectory( $this->source, $translations, $options );
 			}
 
+			if ( ! $this->skip_php ) {
+				$options = [
+					// Extract 'Title' headers from pattern files.
+					'wpExtractPatterns' => isset( $this->main_file_data['Theme Name'] ),
```

Comment:
`isset( $this->main_file_data['Theme Name'] )` this means we only look for patterns in a "theme" directory: a directory that contains theme metadata in its `style.css`.